src/Pos/Communication/Limits.hs

```diff
@@ -283,6 +284,12 @@ instance MessageLimited (MsgHeaders ssc) where
         return $
             MsgHeaders <$> vectorOf Const.recoveryHeadersMessage headerLimit
 
+-- TODO: Update once we move to CBOR.
+instance MessageLimitedPure MsgSubscribe where
+    msgLenLimit = 0
```

Comment:
Yes it's a bit confusing. I believe this limit describes how many bytes will take in when receiving in a `MsgSubscribe` conversation, which is `0` because the other side must send `Void`. The idea is to have the conversation stay open until it fails or one side shuts down.